Internal error in swagger_spec_validator: AttributeError: module 'jsonschema._validators' has no attribute 'properties_draft4'

````
/home/mathieu/Code/Mozilla/kinto/.venv/lib/python3.6/site-packages/swagger_spec_validator/validator20.py:124: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

instance_resolver = <jsonschema.validators.RefResolver object at 0x7f15679b4860>

    def create_dereffing_validator(instance_resolver):
        """Create a customized Draft4Validator that follows $refs in the schema
        being validated (the Swagger spec for a service). This is not to be
        confused with $refs that are in the schema that describes the Swagger 2.0
        specification.
    
        :param instance_resolver: resolver for the swagger service's spec
        :type instance_resolver: :class:`jsonschema.RefResolver`
    
        :rtype: Its complicated. See jsonschema.validators.create()
        """
        visited_refs = {}
    
        custom_validators = {
            '$ref': _validators.ref,
>           'properties': _validators.properties_draft4,
            'additionalProperties': _validators.additionalProperties,
            'patternProperties': _validators.patternProperties,
            'type': _validators.type_draft4,
            'dependencies': _validators.dependencies,
            'required': _validators.required_draft4,
            'minProperties': _validators.minProperties_draft4,
            'maxProperties': _validators.maxProperties_draft4,
            'allOf': _validators.allOf_draft4,
            'oneOf': _validators.oneOf_draft4,
            'anyOf': _validators.anyOf_draft4,
            'not': _validators.not_draft4,
        }
E       AttributeError: module 'jsonschema._validators' has no attribute 'properties_draft4'

/home/mathieu/Code/Mozilla/kinto/.venv/lib/python3.6/site-packages/swagger_spec_validator/ref_validators.py:52: AttributeError

During handling of the above exception, another exception occurred:

tp = <class 'swagger_spec_validator.common.SwaggerValidationError'>, value = None, tb = None

    def reraise(tp, value, tb=None):
        try:
            if value is None:
                value = tp()
            if value.__traceback__ is not tb:
>               raise value.with_traceback(tb)


```

--- a/dev-requirements.txt
+++ b/dev-requirements.txt
@@ -8,6 +8,7 @@ pytest-sugar==0.9.2
 pytest-watch==4.2.0
 SQLAlchemy==1.2.14
 python-memcached==1.59
+swagger-spec-validator==2.4.1
 tox==3.5.3
 WebTest==2.0.32
 werkzeug==0.14.1

